spec.yaml

```diff
@@ -1174,6 +1173,22 @@ paths:
         }
         ```
 
+        ### Example request body for Azure
+
+        ```json
+        {
+          "provider": "azure",
+          "azure": {
+            "credential": {
+              "clientID": "c93bf55e-5bf7-4966-ad2b-e6f6e7721d50",
+              "secretID": "c2VjcmV0IHBhc3N3b3JkIGV4YW1wbGUK",
+              "subscriptionID": "b388b7c7-4479-4040-9ac5-1e13edd6b1cd",
+              "tenantID": "3dd2e94a-92ba-434c-99be-32bb65864a99"
```

Comment:
- `clientID` should become `client_id`
- `secredID` should become `secret_id`
- `subscriptionID` -> `subscription_id`
- `tenantID` -> `tenant_id`

Reply:
fixed